repos/repo_url.go

```diff
@@ -74,11 +78,14 @@ func (r *RepoURL) ValidGitHubURL() error {
 	case "github.com":
 		break
 	default:
-		return fmt.Errorf("%w: %s", ErrorUnsupportedHost, r.Host)
+		//nolint:wrapcheck
+		return sce.Create(ErrorUnsupportedHost, r.Host)
```

Comment:
a bunch of errors (e.g., `ErrorUnsupportedHost `) are declared at https://github.com/ossf/scorecard/blob/c02df073a9f39e6914d7f934eb9f5af131ff2361/repos/repo_url.go#L27:L32

They are also used in a cron job file `cron/data/iterator_test.go`.

We have the main `errors/` directory with error definitions used by checks.

I think we need to start a thread on how to harmonize everything?

Reply:
Ack.